GameOfLife/Logic/DataSerialization.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Runtime.Serialization.Formatters.Binary;
+
+namespace GameOfLife
+{
+    /// <summary>
+    ///  Wraper to encapsulate serialization code
+    /// </summary>
+    public class DataSerialization
+    {
+        /// <summary>
+        /// Serializes file into file path
+        /// </summary>
+        /// <param name="savedData">objects to serialize</param>
+        /// <param name="filePath">relative location of seriliazation file</param>
+        public T? BinarySerialize<T>(T savedData, string filePath)
+        {
+
+            if (File.Exists(filePath))
+            {
+                File.Delete(filePath);
+            }
+
+            FileStream fileStream;
+            BinaryFormatter formatter = new BinaryFormatter();
+
+            using (fileStream = File.Create(filePath))
+            {
+                formatter.Serialize(fileStream, savedData);
+                fileStream.Close();
```

Comment:
using statement will close your stream for you, so you dont need this line anymore. Same in `BinaryDeserialize` method

Reply:
Deleted unnecessary code